en/publish/publish-wechatgame-sub-domain.md

```diff
@@ -1,170 +1,170 @@
-# 接入微信小游戏的开放数据域
+# Access to the open data domain of WeChat games
 
-微信小游戏为了保护其社交关系链数据，增加了 **开放数据域** 的概念，这是一个单独的游戏执行环境。开放数据域中的资源、引擎、程序，都和主游戏完全隔离，开发者只有在开放数据域中才能访问微信提供的 wx.getFriendCloudStorage() 和 wx.getGroupCloudStorage() 两个 API，用于实现一些例如排行榜的功能。由于开放数据域只能在离屏画布 sharedCanvas 上渲染，因此需要我们把 sharedCanvas 绘制到主域上。
+In order to protect its social relationship chain data, **WeChat Games** has added the concept of **Open Data Domain**, which is a separate game execution environment. The resources, engines, and programs in the open data domain are completely isolated from the main game. Developers can access the two `wx.getFriendCloudStorage()` and `wx.getGroupCloudStorage()` APIs provided by WeChat only in the open data domain. For example, the function of the leaderboard. Since the open data field can only be rendered on the offscreen canvas sharedCanvas, we need to draw the sharedCanvas onto the main domain.
```

Comment:
1. WeChat Games -> WeChat Mini Games
2. Open Data Domain -> Open Data Content
3. Developers can access the `wx.getFriendCloudStorage()` and `wx.getGroupCloudStorage()` two APIs provided by WeChat only in the open data content to implement some features, such as leaderboards.
4. Because Open Data Content can only be rendered on off-screen canvas **sharedCanvas**, we need to draw **sharedCanvas** to the main content.

Reply:
ok